src/components/Submitted.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+const Submitted = () => {
+  const navigate = useNavigate();
+  
+  const handleSubmit = () => {
+    navigate(-1);
+  };
+
+  return (
+    <div>
+      <p className="submit-success">Post added!</p>
+
+      <button onClick={handleSubmit}>Return to review</button>
+    </div>
+  );
+};
+
+export default Submitted;
```

Comment:
Nice feedback to the user here that the post has been successful. As discussed previously good implementation of useNavigate to take you to the confirmation page.